docroot/sites/config.setttings.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+// SimpleSAMLphp configuration
+// Provide universal absolute path to the installation.
+if (isset($_ENV['AH_SITE_NAME']) && is_dir('/var/www/html/' . $_ENV['AH_SITE_NAME'] . '/simplesamlphp')) {
+  $settings['simplesamlphp_dir'] = '/var/www/html/' . $_ENV['AH_SITE_NAME'] . '/simplesamlphp';
+}
+else {
+  // Local SAML path.
+  if (is_dir(DRUPAL_ROOT . '/../simplesamlphp')) {
```

Comment:
Instead of hard wiring local config it is better to create a local.settings.php file that is not tracked in the repo instead. I'd remove the local config and cadd an include file statement.

Reply:
ooh i found this. handy little deal https://github.com/acquia/blt/blob/9.x/settings/blt.settings.php#L238